lib/knapsack_pro/runners/queue/minitest_runner.rb

```diff
@@ -32,7 +32,15 @@ def self.run(args)
             accumulator = run_tests(accumulator)
           end
 
+          puts '+'*100
+          puts "Ensure we return exit code: #{accumulator[:exitstatus]}"
+          puts '+'*100
+
           Kernel.exit(accumulator[:exitstatus])
+
+          puts '+'*100
+          puts "Kernel.exit called"
```

Comment:
This should never be printed because when we do `Kernal.exit` then the process will be closed.